developerportal/apps/events/templates/event.html

```diff
@@ -1,57 +1,73 @@
-{% endblock %}
+{% load wagtailimages_tags %}
 
 {% block body_class %}event{% endblock %}
 
 {% block content %}
-  <main>
-    {% include "organisms/article-header.html" with title=page.title description=page.description type="event" %}
-    <section class="section">
-      <div class="mzp-l-content">
-        {% include "molecules/event-details.html" with resource=page %}
-      </div>
-    </section>
-    <section class="section">
-      <div class="mzp-l-content article-content">
-        <div class="article-body">
-          <div class="resource-toolbar">
-            {% include "molecules/resource-share.html" %}
-          </div>
-        </div>
-        <article class="article-body">
-          {% for block in page.body %}
-            {% if block.block_type == 'image' %}
-              {% include "molecules/image-block.html" with block=block %}
-            {% else %}
-              {{ block }}
-            {% endif %}
-          {% endfor %}
-          {% if page.register_url %}
-            <a href="{{ page.register_url }}" class="mzp-c-button">
-              Register now
-            </a>
-          {% endif %}
-          {% include "organisms/event-map.html" with resource=page %}
-          {% include "organisms/event-agenda.html" with resource=page %}
-        </article>
-        <aside class="article-sidebar">
-          {% if page.primary_topic %}
-            {% include "molecules/card-topic.html" with topic=page.primary_topic %}
-          {% endif %}
-        </aside>
+
+{% if page.card_image %}
+  {% image page.card_image width-480 as img %}
+{% else %}
+  {% if page.image %}
+    {% image page.image width-480 as img %}
+  {% else %}
+    {% static "img/placeholders/event.jpg" as fallback_url %}
+  {% endif %}
+{% endif %}
+
+{% static "img/icons/events.svg" as page_icon_asset_url %}
+{% include "molecules/header-strip.html" with page_icon_asset_url=page_icon_asset_url page=page content=page.title element="h1" standfirst=page.summary_meta %}
+
+
+<div class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-right content-page">
+  <main role="main" class="mzp-l-main custom-width">
+      {% for block in page.body %}
+        {% if block.block_type == 'image' %}
+          {% include "molecules/image-block.html" with block=block %}
+        {% else %}
+          {{ block }}
+        {% endif %}
+      {% endfor %}
+
+      <div class="outbound-links">
+        {% if page.register_url %}
+          <a href="{{page.register_url}}" class="mzp-c-button">
+            Register
+          </a>
+        {% endif %}
+        {% if page.official_website %}
+          <a href="{{page.register_url}}" class="mzp-c-button">
+            Official Website
+          </a>
+        {% endif %}
+        {% if page.event_content %}
+          <a href="{{page.register_url}}" class="mzp-c-button">
+            Event Content
+          </a>
+        {% endif %}
```

Comment:
Seems like this is a candidate for an unordered list?

Reply:
Good call - will make the change as part of a mop-up ticket, if that's ok. Noted in #1233

Reply:
Thanks for the pointer about EOL @schalkneethling! Noted in the mop-up ticket

Reply:
Done!